lib/snowpack/type_parser.ex

```diff
@@ -42,6 +42,10 @@ defmodule Snowpack.TypeParser do
 
   defp parse({:integer, data}) when is_binary(data), do: String.to_integer(data)
 
+  defp parse({:array, :null}), do: []
+  defp parse({:array, data}), do: Jason.decode!(data)
+
+  defp parse({:json, :null}), do: :null
```

Comment:
Should this return something that is parseable as json like `{}`?

Reply:
Similar to how you return an empty array.

Reply:
Fair point. That will probably be what any uses of `OBJECT` would wanna use

Reply:
Changed to `{}`